scenario/context/CometContext.ts

```diff
@@ -143,6 +144,36 @@ export class CometContext {
     return this;
   }
 
+  async updatePriceFeeds(newPrices: Record<string, number>) {
+    const comet = await this.getComet();
+    const baseToken = await comet.baseToken();
+
+    const newPriceFeeds: Record<string, string> = {};
+    for (const asset in newPrices) {
+      // XXX use the correct alias
```

Comment:
What alias?

Reply:
like `weth:priceFeed`

Reply:
isn't that what it's doing now?

Reply:
no, right now it's using the asset address, not the alias (e.g. `0x123...:priceFeed`), which is pretty useless